### PR TITLE
New navigation

### LINKED ISSUE #932: Header component 3.0
Please see the prototype below:
https://www.figma.com/proto/MxA95ZHK5NqGX4ptFQljRJ/%E2%AD%90-Design-System-3.0?page-id=4157%3A71619&node-id=9200-137204&viewport=1231%2C-11770%2C0.84&t=Oer6vYeG9nuOnxIR-1&scaling=min-zoom&content-scaling=fixed&starting-point-node-id=9200%3A137204

Design mockup: 
https://www.figma.com/design/MxA95ZHK5NqGX4ptFQljRJ/⭐-Design-System-3.0?node-id=9200-137199&t=cPfWxit2gdsczWlw-11

List of changes:

1. More minimalistic 'Critical/Unread/All' issues and filters row

2. Remove the `code` and `dashboard`  toolbar buttons and reduce them to one navigation bar that allows environment selection when in 'Home'

3. Show an environment breakdown of specific scopes when in a specific span scope (only show if more than one exists)

### LINKED ISSUE #935: Remove redundant query info (already appears in header)

### PR DESCRIPTION
**Pull request by kshmidt-digma**
Closes #935 
Closes #932

### CODE REVIEW

File: src/components/Insights/Issues/IssuesFilter/index.tsx [hunk 1]
```diff
@@ -120,22 +184,48 @@ export const IssuesFilter = ({ query, onApply }: IssuesFilterProps) => {
...
             icon={(props: IconProps) => (
               <s.InsightIconContainer>
-                <InsightsIcon {...props} size={12} color="currentColor" />
+                <FourPointedStarIcon {...props} />
+              </s.InsightIconContainer>
+            )}
+            disabled={issueTypesFilterOptions?.length === 0}
+          />
+          <s.FilterCategoryName>Criticality</s.FilterCategoryName>
+          <Select
```
Comment (opoliarush) [inline src/components/Insights/Issues/IssuesFilter/index.tsx:199]: Why  do we need Select for Each if the options ?

File: src/components/Insights/Issues/useIssuesFilters.ts [hunk 2]
```diff
@@ -1,54 +1,60 @@
...
 };
 
-export const useIssuesFilters = ({
-  refreshInterval,
-  query
-}: UseIssuesFiltersProps) => {
-  const [data, setData] = useState<IssuesFiltersData>({
-    issueTypeFilters: []
-  });
+export const useIssuesFilters = () => {
```
Comment (opoliarush) [inline src/components/Insights/Issues/useIssuesFilters.ts:25]: Can we use useFetchData for this hook ?

File: src/components/Insights/useInsightsData.ts [hunk 3]
```diff
@@ -1,177 +1,213 @@
...
-}: UseInsightDataProps) => {
-  const [data, setData] = useState<InsightsData>({
-    insightsStatus: InsightsStatus.LOADING,
-    insights: [],
-    viewMode: ViewMode.INSIGHTS,
-    totalCount: 0
-  });
-  const previousData = usePrevious(data);
-  const [isInitialLoading, setIsInitialLoading] = useState(false);
+export const useInsightsData = () => {
```
Comment (opoliarush) [inline src/components/Insights/useInsightsData.ts:121]: Can we reuse useFetchData?

File: src/components/Navigation/index.tsx [hunk 4]
```diff
@@ -27,58 +26,56 @@ import { IconButton } from "./common/IconButton";
...
-      if (isAlreadyAtScope(codeContext, scope)) {
-        return "Showing asset runtime data";
-      }
-
-      return "Runtime data available for this code. Click to view";
-    }
-  }
-};
+// const hasData = (codeContext?: CodeContext): boolean =>
+//   Boolean(
```
Comment (opoliarush) [inline src/components/Navigation/index.tsx:35]: Can we remove not used code ?

**Review by opoliarush — CHANGES_REQUESTED** [review]

File: src/components/Insights/Issues/IssuesFilter/index.tsx (hunk 1 above)
Comment (kshmidt-digma) [reply src/components/Insights/Issues/IssuesFilter/index.tsx:199]: Filters UI implemented according to the new design: https://www.figma.com/design/MxA95ZHK5NqGX4ptFQljRJ/%E2%AD%90-Design-System-3.0?node-id=9262-121138&m=dev

I suppose there will be more options later on

**Review by opoliarush — APPROVED** [review]

**Pull request merged**